composer.json

```diff
@@ -105,6 +105,7 @@
         }
     },
     "scripts": {
-        "post-create-project-cmd": "Sous\\Starter::installTheme"
+        "post-create-project-cmd": "Sous\\Starter::installTheme",
+        "install-theme": "Sous\\Starter::installTheme"
```

Comment:
I'm not sure we need a second composer command here. The theme will get installed when creating the project.

Is there a reason a user would need to run `composer install-theme`?

Reply:
I thought that you couldn't run this command manually: `post-create-project-cmd` but apparently you can now, maybe a composer 2.x thing?  I will remove the duplicate.